ambry-quota/src/main/java/com/github/ambry/quota/capacityunit/AmbryCapacityUnitQuotaEnforcer.java

```diff
@@ -15,19 +15,23 @@
 
 import com.github.ambry.messageformat.BlobInfo;
 import com.github.ambry.quota.QuotaEnforcer;
-import com.github.ambry.quota.QuotaRecommendation;
 import com.github.ambry.quota.QuotaName;
+import com.github.ambry.quota.QuotaRecommendation;
 import com.github.ambry.quota.QuotaSource;
 import com.github.ambry.rest.RestMethod;
 import com.github.ambry.rest.RestRequest;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
 
 /**
  * Implementation of {@link QuotaEnforcer} for Capacity Units of ambry resource.
  */
-// TODO: The current implementation allows all requests without any processing. It needs to be replaced by an implementation that enforces quota.
 public class AmbryCapacityUnitQuotaEnforcer implements QuotaEnforcer {
+  private final static List<QuotaName> SUPPORTED_QUOTA_NAMES =
```

Comment:
we can just get rid of this implementation since all it's funcationality will be covered by the new implementation.